### PR TITLE
Check sinceTimestamp before creating a report

### PR DESCRIPTION
**Pull request by michalsidzej**

### CODE REVIEW

File: packages/backend/test/core/reports/fakes.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export function fakeTokenInfo(token: Partial<TokenInfo>): TokenInfo {
     symbol: 'FKT',
     decimals: 18,
     address: EthereumAddress.random(),
-    sinceTimestamp: new UnixTime(0),
+    sinceTimestamp: new UnixTime(0).add(2, 'days'),
```
Comment (sz-piotr) [inline packages/backend/test/core/reports/fakes.ts:93]: I think this one needs a comment

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/backend/test/core/reports/createReports.test.ts [hunk 2]
```diff
@@ -35,3 +38,87 @@ describe(createReports.name, () => {
     ])
   })
 })
+
+describe(aggregateBalancesPerProject.name, () => {
+  it('discards balances before bridge sinceTimestamp', () => {
```
Comment (antooni) [inline packages/backend/test/core/reports/createReports.test.ts:43]:
```suggestion
  it('skips balances before bridge sinceTimestamp', () => {
```

Comment (antooni) [reply packages/backend/test/core/reports/createReports.test.ts:43]: Do I understand it correctly that there is no discarding, just filtering ?

**Review by antooni — CHANGES_REQUESTED** [review]

File: packages/backend/test/core/reports/createReports.test.ts (hunk 2 above)
Comment (michalsidzej) [reply packages/backend/test/core/reports/createReports.test.ts:43]: Yes, that's right.

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**